docs/data/material/components/autocomplete/CheckboxesTags.js

```diff
@@ -16,17 +16,20 @@ export default function CheckboxesTags() {
       options={top100Films}
       disableCloseOnSelect
       getOptionLabel={(option) => option.title}
-      renderOption={(props, option, { selected }) => (
-        <li {...props}>
-          <Checkbox
-            icon={icon}
-            checkedIcon={checkedIcon}
-            style={{ marginRight: 8 }}
-            checked={selected}
-          />
-          {option.title}
-        </li>
-      )}
+      renderOption={(props, option, { selected }) => {
+        const { key, ...optionProps } = props;
```

Comment:
Decided to destructure in a new line instead of doing it inline so it's clearer for users that the first argument is `props`. No strong opinion so let me know if you'd like it to be shorter.

```
      renderOption={({ key, ...optionProps }, option, { selected }) => {
```

Reply:
Makes sense